### DISCUSSION TITLE
Vanilla GatsbyV5 takes ~1 minute to build

### DISCUSSION THREAD

**Discussion by DavidSabine** [gatsbyjs/gatsby#37310] [Community]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

Gatsby V5, out of the box, takes ~1 minute to build.
`gatsby develop` time-to-interaction is ~1 minute.

This is an issue.

### Reproduction Link

https://github.com/DavidSabine/gatsby5-build-time

### Steps to Reproduce

1. `npm init`
2. `npm install gatsby --save-dev`
3. `gatsby develop`

⌚

### Expected Result

I expect time-to-interaction is comparable to other SSR frameworks: Vitepress? Svelte? Nextjs?

### Actual Result

Time-to-interaction is ~1 minute. That's *before* adding any content or images.

Given that Vite builds comparable site in milliseconds, it's time Gatsbyjs take a new look at developer experience.

Gatsbyjs in 2018 was amazing: hot-reloading, image processing, well-documented APIs. Putting up with 1-2minutes when running `npm run develop` was an easy trade-off.

Gatsbyjs entering 2023 is a different story. The productivity gains offered by Vite, Vitest (compared to Webpack, Jest) are significant. It's difficult to justify starting new projects with Gatsby.

### Environment

```shell
System:
    OS: Linux 5.15 Ubuntu 22.04.1 LTS 22.04.1 LTS (Jammy Jellyfish)
    CPU: (8) x64 Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz
    Shell: 5.1.16 - /bin/bash
  Binaries:
    Node: 18.12.1 - ~/.asdf/installs/nodejs/18.12.1/bin/node
    Yarn: 1.22.19 - /mnt/c/Program Files/nodejs/yarn
    npm: 8.19.2 - ~/.asdf/plugins/nodejs/shims/npm
  npmPackages:
    gatsby: 5.3.2 => 5.3.2 
  npmGlobalPackages:
    gatsby-cli: 5.3.1
```

### Config Flags

none

**Comment by tyhopp** [comment]
Thanks for sharing your feedback. Improving time to develop is a metric we track and do look to improve on. The major upgrade from v4 to v5 actually saw a reduction of ~10 seconds by our metrics, so this isn't a problem specific to the recent major release.

**Reply by DavidSabine** [reply]
Yes, sorry, I didn't mean to suggest a problem with V5. I understand V5 has improved on this metric. Perhaps Webpack has become a crutch?
